capstone/src/main/webapp/script.js

```diff
@@ -47,23 +407,67 @@ async function callPerspective(text, lang, requestedAttributes) {
     method: 'POST',
     headers: {'Content-Type': 'application/json',},
     body: JSON.stringify({text: text, lang: lang, requestedAttributes: requestedAttributes})});
-  const toxicityData = await response.json();
-  loadChartsApi(toxicityData);
+  return await response.json();
 }
 
 /** Loads the Google Charts API */
 function loadChartsApi(toxicityData) {
-  google.charts.load('current', {'packages':['corechart']});
-  google.charts.setOnLoadCallback(function() {drawBarChart(toxicityData);}); 
+	google.charts.load('current', {'packages':['corechart']});
+  google.charts.setOnLoadCallback(function() {drawGeneralChart(toxicityData); }); 
+}
+
+/** Draws a Google BarChart for Datamuse replacement analysis. */
+function drawDatamuseChart(responses, substringForreplacements, toxicityOfOriginal, replacements) {
```

Comment:
nit: Let's use camel case for `substringForreplacements` -> `substringForReplacements`

Reply:
Fixed